cli/src/main/java/com/devonfw/tools/ide/tool/pgadmin/PgAdminUrlUpdater.java

```diff
@@ -0,0 +1,55 @@
+package com.devonfw.tools.ide.tool.pgadmin;
+
+import com.devonfw.tools.ide.os.OperatingSystem;
+import com.devonfw.tools.ide.url.model.folder.UrlVersion;
+import com.devonfw.tools.ide.url.updater.WebsiteUrlUpdater;
+import com.devonfw.tools.ide.version.VersionIdentifier;
+
+import java.util.regex.Pattern;
+
+/**
+ * {@link WebsiteUrlUpdater} for pgadmin.
+ */
+public class PgAdminUrlUpdater extends WebsiteUrlUpdater {
```

Comment:
This is always that last resort if no other option is available.
However, I could also not find a better solution.
FYI: https://www.postgresql.org/docs/current/git.html